src/assets/img/icons/mediatype/account.ts

```diff
@@ -1,14 +1,13 @@
 import { svg } from 'lit';
 
 export const accountIcon = svg`
-  <svg viewBox="0 0 100 100"
-    xmlns="http://www.w3.org/2000/svg">
+  <svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+    <title>icon/account</title>
```

Comment:
blocker: We don't know what the `<title>`s should be. And there are conflicting styles.

Let's not make any changes in these, but back out the changes in all the mediatype icons and wait for the right design.

There needs to be a design ticket for this. I will write it.

Reply:
Let's try to center each pull request on the (hopefully) one issue of each Jira ticket. 
Here: "Item tile icon stats color"

If we do get Jira tickets are not clear and focused, let's all include in our development process: sending them back, i.e. commenting that the writer of the ticket needs to 
- be clearer
- break up the ticket
- etc.